lib/flowmailer_rails/mailer.rb

```diff
@@ -38,8 +39,7 @@ def submit_message(json)
       begin
         response = api_client.post(path, json, authorization_header)
 
-        raise ExpiredAccessTokenError if response.status == 401
-        raise DeliveryError, response.body unless response.success?
+        handle_failure(response) unless response.success? && response.status < 400
```

Comment:
`response.success?` will never return true if status is >= 400, so we can remove `&& response.status < 400`

super nitpick: I think it might be cleaner if we remove the `unless response.success?` as well, and then add a `return if response.success?` in the beginning of  the `handle_failure` method 🤷